### PR TITLE
Replace tokio with async-std & add load test

### PR DESCRIPTION
**Pull request by feymartynov**

### CODE REVIEW

File: examples/src/main.js [hunk 1]
```diff
@@ -12,14 +12,26 @@ const JANUS_AGENT_ID = `alpha.${JANUS_ACCOUNT_ID}`;
...
 const PUBLISH_TOPIC = `agents/${JANUS_AGENT_ID}/api/v1/in/${ME_ACCOUNT_ID}`;
 const SUBSCRIBE_TOPIC = `apps/${JANUS_ACCOUNT_ID}/api/v1/responses`;
+const EVENTS_TOPIC = `apps/${JANUS_ACCOUNT_ID}/api/v1/events`;
+
 const PLUGIN = 'janus.plugin.conference';
 const STREAM_ID = '3fdef418-15d3-11ea-9005-60f81db6d53e';
 const CONSTRAINTS = { audio: true, video: { width: 1280, height: 720 } };
 const BUCKET = 'origin.webinar.beta.example.org';
 
+const ICE_SERVERS = [
```
Comment (manifest) [inline examples/src/main.js:24]: One can use open stun server instead: `stun.l.google.com:19302`.

File: src/message_handler/generic/mod.rs [hunk 2]
```diff
@@ -44,10 +41,9 @@ where
     S: 'static + Clone + Send + Sync + Sender,
 {
     pub fn new(sender: S) -> Self {
-        let (tx, rx) = mpsc::sync_channel(10);
+        let (tx, rx) = async_std::channel::bounded(1000);
```
Comment (manifest) [inline src/message_handler/generic/mod.rs:44]: Should we move the channel capacity to the app config?

File: src/message_handler/generic/mod.rs [hunk 3]
```diff
@@ -57,17 +53,18 @@ where
 
     pub fn start(&self) {
         let tx = self.tx.to_owned();
+        let rx = &self.rx;
         let sender = self.sender.to_owned();
         let handler: MessageHandler<S> = MessageHandler::new(tx, sender);
 
         loop {
-            match self.rx.recv().ok() {
+            match async_std::task::block_on(async { rx.recv().await.ok() }) {
```
Comment (manifest) [inline src/message_handler/generic/mod.rs:61]: Why do we call `block_on` in the loop?

File: src/message_handler/generic/mod.rs [hunk 4]
```diff
@@ -57,17 +53,18 @@ where
...
                 None => (),
                 Some(Message::Request(operation, request)) => {
                     verb!("Scheduling request handling");
                     let handler = handler.clone();
 
-                    self.runtime.spawn(async move {
+                    async_std::task::spawn(async move {
```
Comment (manifest) [inline src/message_handler/generic/mod.rs:67]: It would be less verbose to use just `task::spawn`

**Review by manifest — CHANGES_REQUESTED** [review]

File: src/message_handler/generic/mod.rs (hunk 2 above)
Comment (feymartynov) [reply src/message_handler/generic/mod.rs:44]: I think there's no need to configure this value. This has to be just some protective big number to not to create unbounded channel.

File: src/message_handler/generic/mod.rs [hunk 5]
```diff
@@ -57,25 +54,28 @@ where
...
-
-                    self.runtime.spawn(async move {
-                        handler.handle_request(operation, request).await;
-                    });
-                }
-                Some(Message::Response(response)) => {
-                    handler.handle_response(response);
+        task::block_on(async {
+            loop {
+                match rx.recv().await.ok() {
```
Comment (manifest) [inline src/message_handler/generic/mod.rs:63]: What the purpose `.ok()` here?

**Review by manifest — APPROVED** [review]

**Pull request merged**